<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineItem</class>
 <widget class="QDialog" name="OnlineItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CRoundLabel" name="image">
     <property name="minimumSize">
      <size>
       <width>47</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>47</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="nameLab">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 16px;
color: #FFFFFF;
line-height: 22px;
font-style: normal;</string>
       </property>
       <property name="text">
        <string>用户昵称占位占位占位占位占…</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="room_lab">
         <property name="minimumSize">
          <size>
           <width>38</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>38</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: qlineargradient(x1:1, y1:0, x2:0, y2:0, stop:0 #D197FE, stop:1 #9752ED);
border-radius: 8px;


font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 11px;
color: #FFFFFF;
line-height: 16px;
font-style: normal;</string>
         </property>
         <property name="text">
          <string>房主</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="user_lab">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>37</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 10px;
color: #FFFFFF;
text-align: right;
font-style: normal;

background-image: url(:/images/live_lev.png);
background-position: center;
padding-right: 6px; 
 text-align: right;</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="user_btn">
     <property name="minimumSize">
      <size>
       <width>77</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(128,220,255,0.2);
border-radius: 16px;

font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 14px;
color: #80DCFF;
line-height: 20px;
font-style: normal;</string>
     </property>
     <property name="text">
      <string>1号麦</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/images/live_mic_number.png</normaloff>:/images/live_mic_number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="room_btn">
     <property name="minimumSize">
      <size>
       <width>77</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>77</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(245,151,255,0.2);
border-radius: 16px;
font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 14px;
color: #F597FF;
line-height: 20px;
font-style: normal;</string>
     </property>
     <property name="text">
      <string>在房间</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/images/live_inRoom.png</normaloff>:/images/live_inRoom.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRoundLabel</class>
   <extends>QLabel</extends>
   <header location="global">CRoundLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
